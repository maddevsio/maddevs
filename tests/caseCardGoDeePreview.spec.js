import { mount } from '@vue/test-utils'
import CardGoDeePreview from '@/components/Cases/cards-content/CardGoDeePreview'

describe('cardGoDeePreview', () => {
  let wrapper

  beforeEach(() => {
    wrapper = mount(CardGoDeePreview, {
      propsData: {
        pictureFolder: 'godee',
        fileName: 'application-for-passengers-card',
        fileExtension: 'png',
        mobileImage: false,
        mobileImageName: '',
        alt: 'GoDee: bus transit app for passengers',
      },
    })
  })

  // ------ IMPORTANT ----- //
  it('is a Vue instance', () => {
    expect(wrapper.exists()).toBeTruthy()
  })

  it('renders correctly', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
  // --------------------- //
})
