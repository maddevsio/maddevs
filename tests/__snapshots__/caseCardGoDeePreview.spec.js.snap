// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`cardGoDeePreview renders correctly 1`] = `
<div
  class="card-content card-content_"
>
  <div
    class="card-content_text-wrapper"
  >
    <h4
      class="case_title_h4 card-content_title"
    >
      
    </h4>
     
    <p
      class="case_paragraph card-content_description"
    >
      
      
    
    </p>
  </div>
   
  <div
    class="card-content_image-wrapper card-content_image-wrapper-"
  >
    <picture
      class=""
    >
      <source
        class="image"
        srcset="[object Object] ,[object Object] 2x"
        type="image/webp"
      />
       
      <img
        alt="GoDee: bus transit app for passengers"
        class="image"
        height="0"
        loading=""
        src="[object Object]"
        srcset="[object Object] 2x"
        width="0"
      />
    </picture>
     
    <!---->
  </div>
</div>
`;
